### DISCUSSION TITLE
Issue with selecting model

### DISCUSSION THREAD

**Discussion by KChanduu** [anse-app/anse#35] [General]
Hey, I tried both gpt-4 & gpt-4-0314 but in the usage section of my account(open ai) it does not show as two different models. In both cases, open ai usage says that I was using gpt-4-0314.
Can you please help?

**Comment by QuietRocket** [comment]
I'm observing the same effect on OpenAI's usage page. However, looking through the code and intercepting requests to both the OpenAI and the Anse serverless API endpoints, the model is clearly "gpt-4" in the request payload when selecting GPT-4 from select bar. In other words, Anse doesn't _seem_ to be causing this, and is forwarding the request correctly.

You can test this by disabling "Request with backend" and looking at the requests it sends directly to OpenAI in the devtools network tab.

As for why this is happening, I have no idea. Perhaps they classify "GPT-4" as the latest checkpoint?

**Comment by yzh990918** [comment]
Yean, perhaps this is the official rule set.

**Comment by CNSeniorious000** [comment]
See [Continuous model upgrades](https://platform.openai.com/docs/models/continuous-model-upgrades)
